docs/source/overview.md

```diff
@@ -25,19 +25,58 @@ flowchart LR
+- [Google Cloud Storage](https://en.wikipedia.org/wiki/Google_Cloud_Storage): `gc://bucket_name/object_name`
+- [Azure Blob Storage](https://en.wikipedia.org/wiki/Microsoft_Azure#Storage_services): `az://container_name/blob_name`
+
+##### Cloud Service Configuration and Authentication
 
+```{eval-rst}
 Remote object storage paths which require authentication or other specialized configuration may use cloudpathlib client arguments (`S3Client <https://cloudpathlib.drivendata.org/~latest/api-reference/s3client/>`_, `AzureBlobClient <https://cloudpathlib.drivendata.org/~latest/api-reference/azblobclient/>`_, `GSClient <https://cloudpathlib.drivendata.org/~latest/api-reference/gsclient/>`_) and :code:`convert(..., **kwargs)` (:mod:`convert() <cytotable.convert.convert>`).
+
 For example, remote AWS S3 paths which are public-facing and do not require authentication (like, or similar to, :code:`aws s3 ... --no-sign-request`) may be used via :code:`convert(..., no_sign_request=True)` (:mod:`convert() <cytotable.convert.convert>`).
 ```
 
+Each cloud service provider may have different requirements for authentication (there is no fully unified API for these).
+Please see the [cloudpathlib](https://cloudpathlib.drivendata.org/~latest/) client documentation for more information on which arguments may be used for configuration with specific cloud providers (for example, [`S3Client`](https://cloudpathlib.drivendata.org/stable/api-reference/s3client/), [`GSClient`](https://cloudpathlib.drivendata.org/stable/api-reference/gsclient/), or [`AzureBlobClient`](https://cloudpathlib.drivendata.org/stable/api-reference/azblobclient/)).
+
+##### Cloud Service File Type Parsing Differences
+
+Data sources retrieved from cloud services are not all treated the same due to technical constraints.
+See below for a description of how each file type is treated for a better understanding of expectations.
+
+__Comma-separated values (.csv)__:
+
+CSV files stored on cloud services are read directly from the cloud service.
+
+__SQLite Databases (.sqlite)__:
+
+SQLite databases stored on cloud services are downloaded locally before other CytoTable work is performed.
+This is done to account for differences in how [SQLite's virtual file system (VFS)](https://www.sqlite.org/vfs.html) operates in context with cloud service object storage.
+Large SQLite files stored in the cloud may benefit from explicit local cache specification through a special keyword argument (`**kwarg`) passed through CytoTable to [`cloudpathlib`: `local_cache_dir`](https://cloudpathlib.drivendata.org/~latest/caching/#keeping-the-cache-around).
+This argument helps ensure constraints surrounding temporary local file storage locations do not impede the ability to download or work with the data (for example, file size limitations and periodic deletions outside of CytoTable might be encountered within default OS temporary file storage locations).
```

Comment:
Why isn't this default for CytoTable sqlite files? Is it too difficult or niche to implement?

Reply:
This is a great question, thank you for raising it! This is possible to implement as a default but comes with some assumptions. The "quickest" path here would be to use the `target_path` argument location with a subdirectory to house these temporary files. This would theoretically at least double the storage space required for CytoTable output while CytoTable completes its work. For example, if there were two 40GB SQLite files being used as source data from the cloud, I'd estimate that 160 GB would be needed at a minimum to complete processing (probably with a buffer of +20 GB extra or more). While discovering this challenge I wasn't sure if this would be astounding to users or if there was a better path to take. Either way, worth an issue to explore this a bit more!

Reply:
Created new issue to delve into possible default here: #140